wai-app-static/test/WaiAppEmbeddedTest.hs

```diff
@@ -32,7 +32,7 @@ embSpec = do
                              { requestHeaders = [("If-None-Match", "Etag 1")] }
             assertStatus 304 req
 
-        it "ssIndices works" $ do
+        it "ssIndices work" $ do
```

Comment:
Why did you change this? I think the former grammar makes more sense (`ssIndices` is a singular noun as its own configuration value)